sdk/storage/azure-storage-blob/README.md

```diff
@@ -9,159 +9,175 @@ Blob storage is ideal for:
+service = BlobServiceClient.from_connection_string(conn_str="my_connection_string")
 ```
 
 ## Key concepts
-
-Blob storage offers three types of resources:
-* The storage account
-* A container in the storage account
-* A blob in a container
-
-#### Clients
-
-The Storage Blobs SDK provides four different clients to interact with the Blob Service:
-1. **BlobServiceClient** - this client interacts with the Blob Service at the account level. 
-    It provides operations to retrieve and configure the account properties
-    as well as list, create, and delete containers within the account.
-    For operations relating to a specific container or blob, clients for those entities
-    can also be retrieved using the `get_blob_client` or `get_container_client` functions.
-2. **ContainerClient** - this client represents interaction with a specific
-    container, although that container need not exist yet. It provides operations to create, delete, or
-    configure containers and includes operations to list, upload, and delete blobs in the container.
-    For operations relating to a specific blob, the client can also be retrieved using
-    the `get_blob_client` function.
-3. **BlobClient** - this client represents interaction with a specific
-    blob, although that blob need not exist yet. It provides blob operations to upload, download, delete, 
-    create snapshots, and list blobs, as well as specific operations per blob type.
-4. **LeaseClient** - this client represents lease interactions with a ContainerClient or BlobClient.
-    It provides operations to acquire, renew, release, change, and break leases on the resources.
-
-#### Blob Types
-
+The following components make up the Azure Blob Service:
+* The storage account itself
+* A container within the storage account
+* A blob within a container
+
+The Azure Storage Blobs client library for Python allows you to interact with each of these components through the
+use of a dedicated client object.
+
+### Clients
+Four different clients are provided to to interact with the various components of the Blob Service:
+1. **[BlobServiceClient](https://azure.github.io/azure-sdk-for-python/ref/azure.storage.blob.html#azure.storage.blob.BlobServiceClient)** -
+    this client represents interaction with the Azure storage account itself, and allows you to acquire preconfigured
+    client instances to access the containers and blobs within. It provides operations to retrieve and configure the
+    account properties as well as list, create, and delete containers within the account. To perform operations on a
+    specific container or blob, retrieve a client using the `get_container_client` or `get_blob_client` methods.
+2. **[ContainerClient](https://azure.github.io/azure-sdk-for-python/ref/azure.storage.blob.html#azure.storage.blob.ContainerClient)** -
+    this client represents interaction with a specific container, although that container need not exist yet, and allows
```

Comment:
I had to do a double take on this appositive with "although". Maybe suggest "(whether or not that container exists yet)"

Reply:
@bryevdv How do you feel about simply "(which need not exist yet)"

Reply:
👍